### DISCUSSION TITLE
OMOptim does not start

### DISCUSSION THREAD

**Discussion by 21ire** [OpenModelica/OpenModelica#11025] [Q&A]
Hello everyone!

I'm trying to do an optimization with OMOptim, but when I click on the .exe file in order to open it, nothing happens. I looked for the .dll files that need to be in the folder, and there are all there. I tried versions 1.18 and 1.21 of openmodelica

Can you help me solve this problem, please? Thank you in advance.

**Comment by s-celles** [comment]
See https://github.com/OpenModelica/OpenModelica/issues/4559
it seems than OMOptim is not maintained anymore

**Comment by adrpo** [comment]
Yes, unfortunately we do not have the resources to fix OMOptim. It still compiles but it does not work.

**Reply by mario35to** [reply]
hello, is there any update for OMOptim?

**Reply by casella** [reply]
No, sorry

**Comment by Fitia-UCAS** [comment]
Now is 11.21 2024 and I downloaded OpenModelica to learn. Emmm... , it seems the OMOptim does not work still... It first turns out that I lack the omni block block.dll. But I downloaded them and put them in the C:/Windows/System32. It still doesn't work.

**Comment by casella** [comment]
@arun3688 maybe you can help here?
